cluster-provision/k8s/1.21/manifests/nodeports/monitoring.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+kind: Service
```

Comment:
@rmohr I am wondering if it is better to instead of creating a new file that creates another Prometheus service, we just update the Prometheus service that we already have with the nodePort configuration:
https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/k8s/1.21/manifests/prometheus/prometheus/prometheus-service.yaml

Reply:
Fixed. Renamed. :+1: